installer/roles/kubernetes/tasks/main.yml

```diff
@@ -64,7 +64,7 @@
       template:
         src: postgresql-persistent.yml.j2
         dest: "{{ kubernetes_base_path }}/postgresql-persistent.yml"
-        mode: '0600'
+        mode: "0600"
```

Comment:
Please discard the changes in this file, as they unnecessarily pollute the diff. Other than that, this looks fine to me.